hr_timesheet_holiday/README.rst

```diff
@@ -0,0 +1,97 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================================
+Link holidays to analytic accounts
+==================================
+
+This module adds a relation between the Leave Types and the Analytic Accounts.
+When a Leave Request is granted, the granted days are converted to hours and
+added to the Analytic Account for that Leave Type.
+
+When the leave is revoked, the hours are also removed from the analytic
+account.
+
+The hours added to the Analytic Account are read-only, so you cannot edit them
+directly, just by granting or revoking leave requests.
+
+The hours to input per day is configurable at company level.
+
+Limitations:
+
+- Consider that the work days are Monday to Friday
+
+Contributors
+------------
+
+* Tom Blauwendraat <tom@sunflowerweb.nl>
+* Terrence Nzaywa <terrence@sunflowerweb.nl>
+
+Installation
+============
+
+Just install.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Configure the amount of hours per workday of your company.
+#. Configure Analytic accounts for each Leave Type.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Approve a Leave Request
+#. See the hours added to the Analytic Accounts
+#. Revoke a Leave Request
+#. See the hours removed from the Analytic Accounts
+
+Known issues / Roadmap
+======================
+
+* Hours per workday should be configurable per employee, not only by company.
```

Comment:
if you added the function that can be used to customize it, mention it here. Also mention that this probably should be done using resource calendars